### ISSUE TITLE
 Zero address and its challenge 

### ISSUE THREAD

**Issue by mbaneshi** [ton-blockchain/ton#830]
I believe that utilizing a single zero address to handle requests from all other addresses in TON implementation is currently insufficient, either in terms of quantity 
 or needs some enhancing functionality. Do you think there is a deliberate plan or does it need some adjustment, or am I misunderstanding the situation? As far as I know, we should impose some error, in any application which does not understand how TON works and guide them to study more! We have an enormous address for use, and intentionally or unintentionally imposing high load traffic in just one address within a tiny timeframe which exceeds network capability should simply block and return the `design architecture ` error, but zero address have another story!

**Comment by EmelyanenkoK** [comment]
TON architecture allows to process high load on single address: when incoming message rate is higher than one account can process, extra messages are queued to be processed later.
After optimization (currently in master branch / mainnet) network works fine with at least 50 millions message in the queue to one address (tested in private network, seems like limit is much-much higher).

Another challenge is that if you are in the same shard with account with huge queue you may experience delay in receiving messages. There are a few possible workarounds here: allowing deeper sharding, "per account virtual sharding" (processing messages to one account with separate queue), etc.

So, to summarize, TON's architecture is fine, fully capable of this type of load and no special errors or prohibitions are needed. At the same time, some further improvement that facilitate coexistence of different type of loads are possible and researched.
